system/otherLoadTest/playlist.xml

```diff
@@ -342,6 +342,11 @@
 				<impl>openj9</impl>
 				<version>19+</version>
 			</disable>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/15250</comment>
+				<impl>ibm</impl>
+				<version>19+</version>
```

Comment:
19 is out of support. Please set `21+`

Reply:
Suggested changes are done and verified. We can see test is getting disabled on z/OS.

Verification job: Grinder_CR/21853/

DISABLED test targets:
19:52:18  	HCRLateAttachWorkload_previewEnabled_0
19:52:18  	HCRLateAttachWorkload_previewEnabled_1
19:52:18  	HCRLateAttachWorkload_previewEnabled_2